aviary/utils/test/test_aviary_values.py

```diff
@@ -84,34 +84,38 @@ def test_aircraft(self):
         except:
             self.fail('Expecting to be able to set the value of an Enum.')
 
-        try:
-            vals.set_val(Aircraft.Engine.TYPE, 'turbojet')
-            self.assertTrue(vals.get_val(Aircraft.Engine.TYPE)
-                            is GASPEngineType.TURBOJET)
-        except:
-            self.fail('Expecting to be able to set the value of an Enum from a string.')
-
-        try:
-            vals.set_val(Aircraft.Engine.TYPE, 'TURBOJET')
-            self.assertTrue(vals.get_val(Aircraft.Engine.TYPE)
-                            is GASPEngineType.TURBOJET)
-        except:
-            self.fail('Expecting to be able to set the value of an Enum from a string.')
+        # TODO - When we moved the aviary_options into individual component options,
+        # we lost the ability to set them as strings.
```

Comment:
are we going to resolve this as part of this PR? Or just make an issue and do a follow on?

Reply:
The new setter makes this problem go away, so I have removed the comment.